### PR TITLE
New release process using docker, circleci and a centralized building tool

### LINKED ISSUE #22: Docker container versioning not implemented
Hi,
I am using blackbox_exporter and I would like to use the official prom/blackbox-exporter container from docker hub.
The versioning system is not implemented and currently prom/blackbox-exporter:latest contains blackbox-exporter 0.1.0. Is it possible to push containers with a sane versioning system?

### PR DESCRIPTION
**Pull request by sdurrheimer**
Fixes #22

### CODE REVIEW

File: .promu.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    path: github.com/prometheus/blackbox_exporter
+build:
+    flags: -a -tags netgo
+    ldflags: |
+        -X {{repoPath}}/vendor/github.com/prometheus/common/version.Version={{.Version}}
+        -X {{repoPath}}/vendor/github.com/prometheus/common/version.Revision={{.Revision}}
+        -X {{repoPath}}/vendor/github.com/prometheus/common/version.Branch={{.Branch}}
+        -X {{repoPath}}/vendor/github.com/prometheus/common/version.BuildUser={{user}}@{{host}}
+        -X {{repoPath}}/vendor/github.com/prometheus/common/version.BuildDate={{date "20060102-15:04:05"}}
+tarball:
```
Comment (brian-brazil) [inline .promu.yml]: The blackbox.yml should also be included

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,7 @@
-FROM        sdurrheimer/alpine-golang-make-onbuild
+FROM        quay.io/prometheus/busybox:latest
 MAINTAINER  The Prometheus Authors <prometheus-developers@googlegroups.com>
 
-EXPOSE      9115
+COPY blackbox_exporter /bin/blackbox_exporter
```
Comment (brian-brazil) [inline Dockerfile]: And we'll need the config here too

Comment (sdurrheimer) [inline Dockerfile]: Yes of course, sorry, I have been a little too fast

File: .promu.yml (hunk 1 above)
Comment (sdurrheimer) [inline .promu.yml]: Done

**Comment by grobie** [comment]
:+1:

**Comment by sdurrheimer** [comment]
@brian-brazil A final check from you and we are good to go.

**Pull request merged**